### ISSUE TITLE
Provide configuration options to pylint

### ISSUE THREAD

**Issue by assafmendelson** [Microsoft/PTVS#817]
Currently there is only an option to run pylint but we cannot configure it. 
It would be nice to provide means to set the configuration (preferably per project) in the project configuration (i.e. set which messages to show for example). 
It would also be nice to configure some of the messages (e.g. there is a warning if a line is over 100 characters, I might want to change it to 120 characters).
Lastly, it would be very useful to configure means to ignore specific warnings. For example lets say pylint gives a false positive on a line, I would like to mark it somehow so it would ignore it in the future but not ignore the specific error in other places.

**Comment by int19h** [comment]
You should be able to use pylintrc for this, placing it in the appropriate location (i.e. typically at the top level of your project) as described here:

http://docs.pylint.org/run.html

To disable warnings for a specific function or line, you can use specially formatted comments:

http://docs.pylint.org/faq.html#is-it-possible-to-locally-disable-a-particular-message

**Comment by alexranaldi** [comment]
I am also using PyLint from within PTVS and while I can configure it manually, having the configuration available through the VS GUI would be a nice feature (but it's certainly not esssential).

**Comment by int19h** [comment]
Yep. Given that there can be more than one pylintrc (e.g. you can have different rules for different folders), the best way to implement it would be to make a custom editor for pylintrc, that you could open simply by double-clicking the one that you want to edit in Solution Explorer. And also add it as a new template in Add New Item project dialog.
